### PR TITLE
Warn when IPFS is running out of storage

### PR DESCRIPTION
**Pull request by WVerlaek**
## Description

Add alert (warning) when IPFS is close to running out of storage (~few days before actually running out of storage).

Runbook: https://github.com/gitpod-io/runbooks/pull/419

<details>
<summary>Summary generated by Copilot</summary>

### <samp>🤖 Generated by Copilot at 63fc751</samp>

Add a Prometheus alert rule for IPFS storage usage in workspace clusters. The rule warns when the IPFS device has less than 600 GB of free space and provides a runbook URL for troubleshooting.

</details>

## Related Issue(s)

Fixes ENG-556

## How to test

Alert query in explore: [link](https://grafana.gitpod.io/explore?orgId=1&left=%7B%22datasource%22:%22P4169E866C3094E38%22,%22queries%22:%5B%7B%22refId%22:%22C%22,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22P4169E866C3094E38%22%7D,%22editorMode%22:%22code%22,%22expr%22:%22min%28node_filesystem_avail_bytes%7Bdevice%3D%5C%22%2Fdev%2Fmapper%2Flvm--disk-ipfs%5C%22,%20node%3D~%5C%22services-.%2A%5C%22,%20cluster%21~%5C%22ephemeral.%2A%5C%22%7D%29%20by%20%28cluster%29%20%3C%20600%2A1024%2A1024%2A1024%22,%22legendFormat%22:%22__auto%22,%22range%22:true,%22instant%22:true%7D%5D,%22range%22:%7B%22from%22:%22now-30d%22,%22to%22:%22now%22%7D%7D)

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: operations/observability/mixins/workspace/rules/central/ipfs.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    prometheus: k8s
+    role: alert-rules
+  name: ipfs-monitoring-rules
+spec:
+  groups:
+    - name: ipfs-alerts
+      rules:
+        - alert: IPFSAlmostOutOfStorage
+          labels:
+            severity: warning
```
Comment (Furisto) [inline operations/observability/mixins/workspace/rules/central/ipfs.yaml:18]: Warnings tend to be overlooked. We should add another (critical) alert when we fall below 200 GB.

Comment (WVerlaek) [reply operations/observability/mixins/workspace/rules/central/ipfs.yaml:18]: good point

Comment (WVerlaek) [reply operations/observability/mixins/workspace/rules/central/ipfs.yaml:18]: added

**Review by Furisto — APPROVED** [review]

**Pull request merged**